tools/cli/internal/openapi/openapi.go

```diff
@@ -97,11 +97,19 @@ func NewChangelog(base, revision, exceptionFilePath string) (*Changelog, error)
 
 	changelogConfig := checker.NewConfig(
 		checker.GetAllChecks()).WithOptionalChecks(breakingChangesAdditionalCheckers).WithDeprecation(deprecationDaysBeta, deprecationDaysStable)
+
 	return &Changelog{
 		Base:              baseSpec,
 		Revision:          revisionSpec,
 		ExceptionFilePath: exceptionFilePath,
 		Config:            changelogConfig,
+		OasDiff: &OasDiff{
+			base:     baseSpec,
+			external: revisionSpec,
+			config: &diff.Config{
```

Comment:
similar to breaking changes, we need to keep the same config so that we classify all changes the same
Reference:
https://github.com/mongodb/openapi/blob/97a837768a48e1b562a9b9be8232126aec05f783/tools/cli/internal/openapi/openapi.go#L99


should we then use changelogConfig here?

Reply:
those are two different config, the changelog uses `*checker.Config` and diff use `*diff.Config`

Reply:
oh got it! then we need to add the optional checks to the diff as well if possible (if not feel free to resolve)

Reply:
no, there is no option for optionals checks in the Diff. I am using the same Diff.config we use for the merge command

this is the config for the Diff
```go
/ Config includes various settings to control the diff
type Config struct {
	PathFilter              string
	FilterExtension         string
	PathPrefixBase          string
	PathPrefixRevision      string
	PathStripPrefixBase     string
	PathStripPrefixRevision string
	ExcludeElements         utils.StringSet
	IncludePathParams       bool
}
```